README.md

```diff
@@ -16,15 +18,15 @@ For scraping, the script uses simple gnu utils like sed, awk, paste, cut.
 
 ## Requirements
 
-* [webtorrent](https://webtorrent.io/) - A tool to stream torrent. `npm install webtorrent-cli -g`
+* [webtorrent](https://webtorrent.io/) - A tool to stream torrent. `npm install webtorrent-cli -g` or `yay -S webtorrent-cli` for Arch Users
 
 ## Installation
 
 ### cURL
 cURL **notflix** to your **$PATH** and give execute permissions.
 
 ```sh
-$ sudo curl -sL "https://raw.githubusercontent.com/Bugswriter/notflix/master/notflix" -o /usr/local/bin/notflix
+$ sudo curl -sL "https://raw.githubusercontent.com/Edesem/no-emoji-and-porn-notflix/master/notflix" -o /usr/local/bin/notflix
```

Comment:
I am pretty sure if you want this to get merged in, you don't want your fork with these 'cringy' words in the download link.

Reply:
Yeah this was just for anyone who wanted to download my fork instead, so they wouldn't accidentally end up downloading the original one instead. For future reference (this was my first PR), is there a way to just include a single file, or possibly only certain changes done to the fork instead of the entire thing?